lib/hyrax/configuration.rb

```diff
@@ -373,6 +373,12 @@ def subject_prefix
       @subject_prefix ||= "Contact form:"
     end
 
+    attr_writer :extract_full_text
```

Comment:
Good question. Following convention here, so I haven't tested otherwise but can. This would have wider implications on the file as it's the pattern used throughout.